### DISCUSSION TITLE
Twitter OAuth

### DISCUSSION THREAD

**Discussion by SammySergi** [nuxt-community/auth-module#1476] [Ideas & Feature Requests]
### What problem does this feature solve?

I have had an abundance of success adding Google, GitHub, and Facebook support to my user authentication system. I was having difficulties hooking into Twitter's authentication flow myself and was wondering if there will be any official support for Twitter logins and authorisation.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/auth-module/issues/c446">#c446</a>)</em></sub></div>

**Comment by frizurd** [comment]
+1

**Comment by phani1kumar** [comment]
+1

**Comment by bmulholland** [comment]
PRs gladly accepted :)

**Comment by bmulholland** [comment]
Also: Please do not use a "+1" comment to indicate support. Use the reaction feature instead. Doing so does not send out notifications, and is what I use to sort issues by to indicate popularity.

**Comment by joshtom** [comment]
@ThatKawaiiSam were you able to get the twitter Oauth to work?

**Comment by suleymangezsat** [comment]
Here is my solution for Twitter OAuth : https://codesandbox.io/s/eloquent-field-3ihhzh
Note: since requests to Twitter go through my backend, my backend has its own jwt mechanism (local scheme).
